README.md

```diff
@@ -118,6 +118,25 @@ curl http://localhost:8080/products/read/1
 curl http://localhost:8080/orders/read/1
 ```
 
+### Search
+Now, search for users
+
+```shell
+curl http://localhost:8080/users/search \                                                                                                                           
+    -X POST \
+    -H 'Content-Type: application/json' \
+    -d '{"q":"John"}'
+```
+
+Or search for products with price less than 50
```

Comment:
Should we add `search query` as well in this case? basically, search query with filtering example.

Reply:
done